spec/components/intervention_spec.rb

```diff
@@ -63,4 +63,65 @@ def component_name
   it "doesn't render anything if no parameter is passed" do
     assert_empty render_component({})
   end
+
+  describe "new tab" do
+    it "renders with target=_'blank' with new_tab option" do
+      render_component(
+        suggestion_link_text: "Travel abroad",
+        suggestion_link_url: "/travel-abroad",
+        new_tab: true,
+      )
+
+      assert_select "a[target='_blank']"
+    end
+
+    it "renders with security attributes" do
```

Comment:
This test appears to do the same steps as the previous test but check for something different. Would it be worth combining the two? i.e. `it renders with target _blank and appropriate security attributes when new_tab is specified`

Reply:
I'd prefer to keep these separate. I feel that testing as specific as possible is a good thing - for clarity more than anything.